Loop/Managers/LoopDataManager.swift

```diff
@@ -1634,6 +1634,34 @@ extension LoopDataManager {
             handler(self, LoopStateView(loopDataManager: self, updateError: updateError))
         }
     }
+    
+    func generateSimpleBolusRecommendation(carbs: HKQuantity?, glucose: HKQuantity?) -> HKQuantity? {
```

Comment:
Unit test for this?

Reply:
This is mainly just passing values, and the IOB component will be removed in an upcoming PR, so I'm going to skip testing this for now.